static/src/stylesheets/layout/new-header/_menu-item.scss

```diff
@@ -107,11 +124,22 @@
         pointer-events: none;
     }
 
+    &[data-link-name='nav2 : the guardian app'] {
+        @include mq($until: desktop) {
+            margin-top: $gs-baseline * 2;
+        }
+    }
+
     &[data-link-name='nav2 : the guardian app'],
-    &[data-link-name='nav2 : facebook'] {
+    &[data-link-name='nav2 : facebook'],
+    &[data-link-name='nav2 : twitter'] {
         @include mq($until: desktop) {
             margin-top: $gs-baseline * 2;
         }
+
+        @include mq(desktop) {
+            display: none;
```

Comment:
Why aren't these relevant on desktop? Is there a reason other than they aren't in the design?